app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -128,25 +128,31 @@ function YourTurnContent(props: {
                     alignItems="center"
                     spacing={2}
                   >
-                    <Grid>
-                      <GreenCheckbox
-                        checked={
-                          shownCardsInActiveTurn.get(cardId) ===
-                          ShownCardStatus.Completed
-                        }
-                        onChange={({ target: { checked } }) => {
-                          setShownCardsInActiveTurn(
-                            new Map(
-                              shownCardsInActiveTurn.set(
-                                cardId,
-                                checked
-                                  ? ShownCardStatus.Completed
-                                  : ShownCardStatus.Incomplete
+                    <Grid item>
+                      <Box>
+                        <GreenCheckbox
+                          checked={
+                            shownCardsInActiveTurn.get(cardId) ===
+                            ShownCardStatus.Completed
+                          }
+                          onChange={({ target: { checked } }) => {
+                            setShownCardsInActiveTurn(
+                              new Map(
+                                shownCardsInActiveTurn.set(
+                                  cardId,
+                                  checked
+                                    ? ShownCardStatus.Completed
+                                    : ShownCardStatus.Incomplete
+                                )
                               )
                             )
-                          )
-                        }}
-                      ></GreenCheckbox>
+                          }}
+                        ></GreenCheckbox>
+                      </Box>
+                      {shownCardsInActiveTurn.get(cardId) ===
+                        ShownCardStatus.Skipped && (
+                        <Box color={grey[500]}>(skip)</Box>
```

Comment:
show if skipped... the word "skipped" is too long to fit here, so using "skip" instead